haproxy/metadata.csv

```diff
@@ -234,3 +232,5 @@ haproxy.server.unsafe.idle.connections.current,gauge,,,,Current number of unsafe
 haproxy.server.used.connections.current,gauge,,,,Current number of connections in use (>= 2.3).,0,haproxy,
 haproxy.server.uweight,gauge,,,,Server's user weight or sum of active servers' user weights for a backend (>= 2.4).,0,haproxy,
 haproxy.server.weight,gauge,,,,Service weight.,0,haproxy,
+haproxy.sticktable.size,gauge,,,,"Maximum number of elements the table can hold (<= 2.3 through unix socket, >= 2.4 through prometheus).",0,haproxy,
+haproxy.sticktable.used,gauge,,,,"Number of elements in the table (<= 2.3 through unix socket, >= 2.4 through prometheus).",0,haproxy,
```

Comment:
```suggestion
haproxy.sticktable.size,gauge,,,,"Maximum number of elements the table can hold (>= 2.4).",0,haproxy,
haproxy.sticktable.used,gauge,,,,"Number of elements in the table (>= 2.4).",0,haproxy,
```
Since we only send the metric exported by prometheus, let's only document that

Reply:
Are you sure? this metric is present while using unix socket.